recipes/xorg-util-macros/bld.bat

```diff
@@ -0,0 +1,7 @@
+:: Just delegate to the Unixy script; but some of the X.org tarballs have
+:: config.guess files too ancient to recognize 64-bit Windows!
+curl -sSL -o config.guess "http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess"
+curl -sSL -o config.sub "http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub"
```

Comment:
Echoing thoughts from this [thread]( https://github.com/conda-forge/staged-recipes/pull/2208#discussion_r96120143 ).

cc @ocefpaf